### ISSUE TITLE
Add 'inhibits' and 'is inhibited by' (from Cyber Ontology)

### ISSUE THREAD

**Issue by cameronmore** [CommonCoreOntology/CommonCoreOntologies#241]
It was recently discovered and discussed at a meeting of the IEEE Cyber Ontology Working Group that the object property `inhibits` caused an inconsistency due to its superproperty chain, that:

If x is cause of y and y has participant r and r is realized in y, and x is cause of d, then x inhibits y.

An example we discussed at a meeting was that if I get sick, and being sick causes me to perform worse in a subsequent sporting event, then the sickness inhibited my performance in the sporting event.

Below is the definition of the current set of object properties:

`inhibits = def. x inhibits y iff x and y are non-identical Processes, d is a Decrease of Realizable Entity, and x is_cause_of d, and r is a Realizable Entity, and d has_participant r, and r realized_in y.`

`inhibited_by = def. y inhibited_by x iff x and y are non-identical Processes, d is a Decrease of Realizable Entity, and x is_cause_of d, and r is a Realizable Entity, and d has_participant r, and r realized_in y.`

 Being sick (x) causes a decrease of realizable entity (d) (suppose, my disposition to run, r), and that disposition (r) is realized in my participation in a sporting event (y), then being sick (x) inhibits the sporting event (y).

We agreed to make move the property under owl:topObjectProperty, but while discussing the term, doubts were raised whether the definition fits under 'disrupts' :

`disrupts = def. A relation where one process disrupts another process from occurring as it would have` with the following elucidation: `A process can disrupt another process from occurring as it would have by 1) preventing a disposition or role from being realized by that process, 2) lowering the grade of the process, or 3) stopping the process from continuing to occur.`

Particularly, there was a question as to whether `...process from occurring as it would have by ...` means that the process must occur but be modified (weakened), or that the process can be stopped from occurring altogether. If the sickness I have prevents me from participating in the race, does the sickness inhibit my performance in the race (if x prevents y, then x stops y from occurring)? Or does it only inhibit if I actually participate? (1) seems to indicate that the process may not occur or that I may not participate in the process, but strictly speaking, I still may participate but not realize my disposition.

I think 'inhibits' fits the definition of 'disrupts', particularly (2) in the elucidation. The definition of inhibits says that the realizable is still realized in the process, but just that there has been a Decrease of Realizable Entity of the realizable which participates.

tldr; 'inhibits' should be a subproperty of 'disrupts' and it is a different discussion whether 'prevents' is an intended or possible interpretation of disrupts. If the definition of 'disrupts' is expanded to include prevention, then 'inhibits' can still remain a subproperty.

Also, the current 'is_curated_in_ontology' for disrupts is incorrect. (It says Quality Ontology but should be Extended Relation Ontology).

@mark-jensen @APCox

**Comment by neilotte** [comment]
An update has been merged into development to address this issue. Closing.
